src/data/option/defs.lean

```diff
@@ -5,6 +5,10 @@ Authors: Mario Carneiro
 -/
 
 /-!
+THIS FILE IS SYNCHRONIZED WITH MATHLIB4.
+https://github.com/leanprover-community/mathlib4/pull/XXX
```

Comment:
Shouldn't you fill in the value of `XXX`?

Reply:
Ugh, thanks :roll_eyes: